WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -33,7 +33,7 @@ private extension DefaultProductFormTableViewModel {
         return actions.map { action in
             switch action {
             case .images(let editable):
-                return .images(isEditable: editable, allowsMultiple: product.allowsMultipleImages())
+                return .images(isEditable: editable, allowsMultiple: product.allowsMultipleImages(), productLabel: product.status.rawValue)
```

Comment:
One thing I wonder here is if this makes sense only for drafts, or also for any status other than publish.

In any case, passing the `ProductStatus` from `product.status` might be more convenient than using the raw value, since you can compare agains `.draft` to check if it's a draft, or use the `description` method to get the label text instead of hardcoding the "Draft" string in the view.

Reply:
> One thing I wonder here is if this makes sense only for drafts, or also for any status other than publish.

That's something that crossed my mind as well. At the time I only added the option for "Draft" in order to replicate how the Android app works, but it would definitely make sense for other statuses and I'm happy to make the changes. I'll be opening a new enhancement request in the Android repo as well to get feedback on this.